From c1b85c3a59ab7b809e9531e1fb4bf38d9476fefc Mon Sep 17 00:00:00 2001
From: Stanislav Brabec <sbrabec@suse.com>
Date: Tue, 22 Nov 2016 14:44:21 +0100
Subject: [PATCH] move privatedir from libdir to libexecdir

According to the GNU Coding Standards[1], private executables should be
installed to libexecdir, not libdir.

Move privatedir to libexecdir.

[1] https://www.gnu.org/prep/standards/

https://bugs.freedesktop.org/show_bug.cgi?id=98817
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 64f0b5a..5cb1f17 100644
--- a/configure.ac
+++ b/configure.ac
@@ -498,7 +498,7 @@ esac
 AC_DEFINE_UNQUOTED(SHLEXT, ["$SHLEXT"], [File extension for shared libraries])
 AC_SUBST(SHLEXT)
 
-privatedir='${libdir}/p11-kit'
+privatedir='${libexecdir}/p11-kit'
 AC_SUBST(privatedir)
 
 AC_CONFIG_FILES([Makefile
-- 
2.10.2

